src/App.js

```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Button from "./components/Button/Button";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Button prop="plus" />
+      <Button prop="minus" />
+      <Button prop="add" text="Add to cart" />
```

Comment:
Instead using `prop` as property is better to tell something more meaningful, like `variant`.

But now in your case, you don't need to have 3 kinds of `variant/prop` you just need 2, because the button `-` and `+` have the same style, so that you can use the same variant and pass a different text, like so:

```jsx
<Button text="+" />
<Button text="-" />
```

These components are generic and what you're looking for is the style and not the meaning. Try something like:

```suggestion
      <Button text="+" />
      <Button text="-" />
      <Button variant="orange" icon="cart" text="Add to cart" />
```

When you don't pass `variant` you can use the default gray button:
![Screen Shot 2022-04-18 at 8 50 42 AM](https://user-images.githubusercontent.com/2808008/163804227-f6588917-0158-47a0-bb1e-a905ee0c064f.png)

if you passed `variant="orange"` you style it for the orange button:
![Screen Shot 2022-04-18 at 8 51 21 AM](https://user-images.githubusercontent.com/2808008/163804296-20db9846-c147-483f-b776-193bf087c2b3.png)